### DISCUSSION TITLE
【委托】大佬们帮帮我！不懂怎么进绳网！

### DISCUSSION THREAD

**Discussion by uyfh352** [share121/inter-knot#987] [其他]
大佬们帮帮我！刚在b站看到一个up弄了绳网，就想着进去玩玩看。但没想到进去那么难啊，搞了一堆东西都不明白。
求大佬帮帮忙，没有委托金，就当做是做善事了
![圖片_20240810202010](https://github.com/user-attachments/assets/b21f09c6-43b2-4445-a59f-5baf63523db2)

**Comment by mwk07** [comment]
教程:1.先下载Steam++，就是简介里那个，浏览器直接搜就行，勾选Github加速。
2.打开github.com/share121/inter-knot，注册登录，登录完一般会跳到自己的页面，再输一次网址就行，网页右边有一个绳网，下面是链接，打开然后它会提醒你安装脚本，点进去，然后先把这个页面放在那里。
3.浏览器搜索tampermonkey，然后找到添加这个浏览器插件就行，然后回到绳网助手页面，点击安装，后面油猴就会跳出来直接安装就行。
4.回到绳网，回不去的还是第二个网址再进一次，它就会要限权，总是允许就行，这样就进去了。（复制的）

**Comment by fu-fu404** [comment]
你返回一下就好了

**Comment by Qiling-XwX** [comment]
首先下载0申
然后0申会自动帮你进入绳网☝🤓

**Comment by justafish8787** [comment]
你这不是已经进了吗_(:з)∠)_
